components/2.0/ComboBox.qml

```diff
@@ -148,7 +151,7 @@ FocusScope {
         id: dropDown
         width: container.width; height: 0
         anchors.top: container.bottom
-        anchors.topMargin: -1
+        anchors.topMargin: 0
```

Comment:
This seems unrelated to borders.

Seems somewhat sensible anyway; make sure you test that with a widget with border 1.

Reply:
I tested the change with a borderwidth of 1 and it looks how it should, for comparison I included a picture with a borderwidth of 5. The entries have smaller space because the overall height remains constant in both cases.

The 'anchors.topMargin: -1' prevented the line between the uppermost text and the list view from having a width of 2 pixel, but due to my code modifications this isn't an issue anymore.

![borderwidth_5](https://user-images.githubusercontent.com/24816369/31275979-dcf3a1a6-aa99-11e7-97b9-3ebdbef1d890.png)
![borderwidth_1](https://user-images.githubusercontent.com/24816369/31275980-dcf7a896-aa99-11e7-8ef1-7ba68e656314.png)